src/BOTIFY_PROJECT_AGGREGATE_URLS.js

```diff
@@ -164,5 +168,20 @@ function BOTIFY_PROJECT_AGGREGATE_URLS(
     }
   }
 
+  // Append errors
+  if (errors) {
+    var errors_list = [];
+    errors_list.length = result[0].length;
+    var have_errors = false;
+    responses[0].forEach(function (response, i) {
+      if (response.error) {
+        have_errors = true;
+        errors_list[start + i] = response.error.message
```

Comment:
I don't quite understand what you are trying to do here, could you please add a small comment with a concrete example :pray: